.github/workflows/xrt_master_2021.1.yml

```diff
@@ -212,6 +219,95 @@ jobs:
           USER: ${{ secrets.USER }}
           github-server-url: ${{ secrets.SERVER_URL }}
 
+      - name: Find and copy packages      
+        run: |      
+          cd ${{ env.PATH_DIR }}/artifact    
+          echo "Listing all files in artifact directory:"  
+          ls -al     
+          # rm -rf xrt_202110.2.11.859_amzn2-x86_64-xrt.rpm  
+          for package_file in *-xrt.rpm *-xrt.deb; do     
+            echo "Found package: $package_file"      
+            cp $package_file ${{ github.workspace }}/${{ env.XRT_VERSION_PATCH }}/Dockerfile/      
+          done      
+    
+          # Copy only the required XCR and XRM packages  
+          base_path="${{ secrets.APP_CONFIG4 }}"  
+          cp $base_path/XCR/master/202220.1.1.27/packages/xcr_202220.1.1.27_18.04-x86_64.deb ${{ github.workspace }}/${{ env.XRT_VERSION_PATCH }}/Dockerfile/  
+          cp $base_path/XCR/master/202220.1.1.27/packages/xcr_202220.1.1.27_20.04-x86_64.deb ${{ github.workspace }}/${{ env.XRT_VERSION_PATCH }}/Dockerfile/  
+          cp $base_path/XCR/master/202220.1.1.27/packages/xcr_202220.1.1.27_22.04-x86_64.deb ${{ github.workspace }}/${{ env.XRT_VERSION_PATCH }}/Dockerfile/  
+          cp $base_path/XCR/master/202220.1.1.27/packages/xcr_202220.1.1.27_7.9.2009-x86_64.rpm ${{ github.workspace }}/${{ env.XRT_VERSION_PATCH }}/Dockerfile/  
+          cp $base_path/XCR/master/202220.1.1.27/packages/xcr_202220.1.1.27_2-x86_64.rpm ${{ github.workspace }}/${{ env.XRT_VERSION_PATCH }}/Dockerfile/  
+    
+          cp $base_path/XRM/master/202220.1.5.103/packages/xrm_202220.1.5.103_18.04-x86_64.deb ${{ github.workspace }}/${{ env.XRT_VERSION_PATCH }}/Dockerfile/  
+          cp $base_path/XRM/master/202220.1.5.103/packages/xrm_202220.1.5.103_20.04-x86_64.deb ${{ github.workspace }}/${{ env.XRT_VERSION_PATCH }}/Dockerfile/  
+          cp $base_path/XRM/master/202220.1.5.103/packages/xrm_202220.1.5.103_22.04-x86_64.deb ${{ github.workspace }}/${{ env.XRT_VERSION_PATCH }}/Dockerfile/  
+          cp $base_path/XRM/master/202220.1.5.103/packages/xrm_202220.1.5.103_8.1.1911-x86_64.rpm ${{ github.workspace }}/${{ env.XRT_VERSION_PATCH }}/Dockerfile/  
+          cp $base_path/XRM/master/202220.1.5.103/packages/xrm_202220.1.5.103_2-x86_64.rpm ${{ github.workspace }}/${{ env.XRT_VERSION_PATCH }}/Dockerfile/  
+          echo "Copied required XCR and XRM packages"  
+        shell: bash    
+        
+      - name: List contents of Dockerfile directory      
+        run: |      
+          echo "Contents of Dockerfile directory:"      
+          ls -al ${{ github.workspace }}/${{ env.XRT_VERSION_PATCH }}/Dockerfile      
+        shell: bash       
+        
+      - name: Build Docker images          
+        run: |          
+          set -e  # Exit on any error        
```

Comment:
Same comment as above. This is a shell script and we should probably have it separated. Would make it:
1. More readable
2. easily callable/testable outside of github actions

Reply:
Updated code for addressing this comment - calling docker build images part from composite workflow